### ISSUE TITLE
Dashboard: Error messages disappear and don't follow WET format

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#35]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard)
User Agent: Chrome, Firefox
Language: English/French

Issue Description: There are a couple of issues with errors for the required fields:

1. Error message for "Alert type" and for "Recall type" do not display properly and hover below the drop down for about 4 seconds and disappear. The error message should be between the label and the input field and should be part of the <label>.
2. After selecting "Recall type" hit Submit. Error message for "Recall title" field does not display on screen for required fields while screen reader announces an error message. Since I don't have access to Create a new alert, I'm not able to test "Alert title" field.
3. The error message for both drop downs is "Please select an item in the list. alert". The word "alert" can be confusing for the user as there is "Create a new alert" section.

Steps to reproduce: 

1. Click on both Submit buttons without filling out any information. 
2. Repeat after selecting an option the first field and leave the 2nd field blank.
3. Repeat above steps with NVDA on as well.

Expected results: Error messages should be displayed clearly and should remain on the screen until user has entered information for the field. The message should be between the label and the input field. 

Please follow WET https://wet-boew.github.io/wet-boew/demos/formvalid/formvalid-en.html

![image](https://user-images.githubusercontent.com/14280062/128245109-61d25b3f-b895-4a2f-9156-b633e1b8a658.png)

![image](https://user-images.githubusercontent.com/14280062/128245906-6be01708-0ee4-46f4-8ddf-3f0e7dc6a3f8.png)

**Comment by davidgotlieb** [comment]
Tooltips are turned off, no longer able to duplicate

**Comment by hydegeoff** [comment]
This issue was fixed in French and English
